Patch by Robert Scheck <redhat@linuxnetz.de> for uw-imap >= 2004a, which corrects
paths to imapd, ipop2d and ipop3d in the man pages. 

This patch is based on Red Hat Bugzilla ID #127271 and solves ID #229781.

--- imap-2004a/src/imapd/imapd.8		2004-05-18 19:46:54.000000000 +0200
+++ imap-2004a/src/imapd/imapd.8.doc		2004-07-23 15:24:17.000000000 +0200
@@ -3,7 +3,7 @@
 .SH NAME
 IMAPd \- Internet Message Access Protocol server
 .SH SYNOPSIS
-.B /usr/etc/imapd
+.B /usr/sbin/imapd
 .SH DESCRIPTION
 .I imapd
 is a server which supports the
--- imap-2004a/src/ipopd/ipopd.8		2004-05-18 19:50:05.000000000 +0200
+++ imap-2004a/src/ipopd/ipopd.8.doc		2004-07-23 15:23:38.000000000 +0200
@@ -3,9 +3,9 @@
 .SH NAME
 IPOPd \- Post Office Protocol server
 .SH SYNOPSIS
-.B /usr/etc/ipop2d
+.B /usr/sbin/ipop2d
 .PP
-.B /usr/etc/ipop3d
+.B /usr/sbin/ipop3d
 .SH DESCRIPTION
 .I ipop2d
 and
